[2025-03-30T16:36:08.575+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-30T16:36:08.598+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.check_data_postgres manual__2025-03-30T16:25:25.435161+00:00 [queued]>
[2025-03-30T16:36:08.608+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.check_data_postgres manual__2025-03-30T16:25:25.435161+00:00 [queued]>
[2025-03-30T16:36:08.609+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-03-30T16:36:08.625+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): check_data_postgres> on 2025-03-30 16:25:25.435161+00:00
[2025-03-30T16:36:08.641+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'weather_data_pipeline', 'check_data_postgres', 'manual__2025-03-30T16:25:25.435161+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpoq9pl_77']
[2025-03-30T16:36:08.643+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask check_data_postgres
[2025-03-30T16:36:08.644+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=860) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-30T16:36:08.645+0000] {standard_task_runner.py:72} INFO - Started process 863 to run task
[2025-03-30T16:36:08.706+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_data_pipeline.check_data_postgres manual__2025-03-30T16:25:25.435161+00:00 [running]> on host 2b1913b6b87b
[2025-03-30T16:36:08.790+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='check_data_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-03-30T16:25:25.435161+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-30T16:25:25.435161+00:00'
[2025-03-30T16:36:08.791+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-30T16:36:08.811+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-30T16:36:08.822+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-30T16:36:08.822+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=check_data_postgres, run_id=manual__2025-03-30T16:25:25.435161+00:00, execution_date=20250330T162525, start_date=20250330T163608, end_date=20250330T163608
[2025-03-30T16:36:08.861+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-30T16:36:08.878+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-30T16:36:08.880+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
